AdobeMediaGallery/Model/Keyword/Command/SaveAssetKeywords.php

```diff
@@ -47,7 +47,11 @@ public function __construct(
     }
 
     /**
-     * @inheritdoc
+     * Save asset keywords.
+     *
+     * @param KeywordInterface[] $keywords
+     * @param int $assetId
+     * @throws CouldNotSaveException
```

Comment:
I'd use `@inheritdoc` here

Reply:
I removed it because of [this rule in the dockbloc standards](https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc) "The @inheritdoc tag SHOULD NOT be used. If a child class method requires a long description to explain its purpose, it may use @inheritdoc to indicate the new description is intended as an addition to the parent method description. In general such method overrides are a code smell and should be used as an incentive to make the code more self-documenting if possible."